docs/guide/cache/content_aware_cache.md

```diff
@@ -293,3 +350,253 @@ bin/console fos:httpcache:invalidate:tag ez-all
+- `https://your-page-blah-blah.us-2.platformsh.site/_fos_user_context_hash` : here we use the hostname we earlier told
+  curl how to resolve using `---resolve`. `/_fos_user_context_hash` is the route to the controller that are able to
+  deliver the user-context-hash.
+- You may also provide the session cookie (`--cookie ".....=....") for a logged-in-user if you are interested in
+  the x-user-context-hash for a different user but anonymous
+
+The output for the given command should look similar to this:
+```
+    HTTP/1.1 200 OK
+    Server: nginx/1.20.0
+    Content-Type: application/vnd.fos.user-context-hash
+    Transfer-Encoding: chunked
+    Connection: keep-alive
+    X-User-Hash: daea248406c0043e62997b37292bf93a8c91434e8661484983408897acd93814
+    Cache-Control: max-age=600, public
+    Date: Tue, 31 Aug 2021 13:35:00 GMT
+    Vary: Origin
+    Vary: cookie
+    Vary: authorization
+    X-Cache-Debug: 1
+    Surrogate-Key: ez-user-context-hash ez-all fos_http_cache_hashlookup-
+```
+
+The header `X-User-Hash` is the one we are interested in here, but you may also note the `Surrogate-Key` which
+holds the [cache tags](#understanding-cache-tags).
+
+### Fetching the HTML response
+
+Now that we have the user-context-hash, we can ask origin for the actual resource we are after:
+
+```bash
+    $ curl -IXGET --resolve www.staging.foobar.com.us-2.platformsh.site:443:1.2.3.4 --header "Surrogate-Capability: abc=ESI/1.0" --header "x-user-hash: daea248406c0043e62997b37292bf93a8c91434e8661484983408897acd93814" https://www.staging.foobar.com.us-2.platformsh.site/
+```
+
+The output :
+```
+HTTP/1.1 200 OK
+Server: nginx/1.20.0
+Content-Type: text/html; charset=UTF-8
+Transfer-Encoding: chunked
+Connection: keep-alive
+Cache-Control: public, s-maxage=86400
+Date: Wed, 01 Sep 2021 07:18:27 GMT
+X-Cache-Debug: 1
+Vary: X-User-Hash
+Vary: X-Editorial-Mode
+Surrogate-Control: content="ESI/1.0"
+Surrogate-Key: ez-all c52 ct42 l2 pl1 p1 p2 r56 r57
+```
+
+The `Cache-Control` header tells the HTTP cache to store the result in the cache for 1 day (86400 seconds)
+The `Vary: X-User-Hash` header tells the HTTP cache that this cache element may be used for all users which has
+the given `x-user-hash` (`daea248406c0043e62997b37292bf93a8c91434e8661484983408897acd93814`).
+The document might also be removed from the cache by purging any of the keys provided in the `Surrogate-Key` header.
+
+So back to the original problem here. This resource is for some reason not cached by Fastly ( remember the
+`x-cache: MISS` we started with). But origin says this page can be cached for 1 day. How can that be?
+The likely reason is that this page also contains some ESI fragments and that one or more of these are not cachable.
+
+So, first let's see if there are any ESIs here. We remove the `-IXGET` options (in order to see content of the response,
+not only headers) to curl and search for esi:
```

Comment:
```suggestion
Coming back to the original problem here. This resource is for some reason not cached by Fastly (see the
`x-cache: MISS` you started with). However origin says this page can be cached for 1 day. 
The reason for this is that this page also contains some ESI fragments and that one or more of these are not cachable.

First, check for any ESIs here. Remove the `-IXGET` options (in order to see content of the response,
not only headers) to curl and search for ESI:
```

Reply:
You removed "likely" from "likely reason". At this time in the debugging session, this is the hypothesis we are going to validate next.

Reply:
I put the word likely back.